### DISCUSSION TITLE
teal dreamboard

### DISCUSSION THREAD

**Discussion by kjloveless** [teal-fm/teal#20] [Ideas]
hi everyone!

we don't have a very centralized place to gather community feedback or ideas/wishes... let's change that!

right now we don't have many concrete plans, other than to bring your music listening data/habits into your hands.
our site lists a few more ideas, but we'd love to hear yours!

**Comment by Hoid** [comment]
Most of my feature ideas are surrounding three things:
1. playlist curation
2. music discovery
3. socialization through music taste

For example, for number 3, I'd really like to see a service that allows you to share what you're listening to on any streaming service to atproto so that other services (like the Bluesky app or other social apps) could display what you're currently listening to, if you allow it. That would require Bluesky to start listening to the firehose for your lexicon and just tracking scrobble events, so whenever it sees an event come across for the song "Mr. Brightside" (which is 3m43s long) it could update its UI to show that you're listening to Mr. Brightside for the next 3 minutes and 43 seconds, unless it receives a new scrobble event.

**Reply by crystal-prison** [reply]
I'd very much like a way to take my lfm favs and make a playlist; i use foobar2k and my internal db is gone since a windows install and its an old version and such. 

So, mostly a foobar2000 plugin of some kind with the possibility of syncing track data i'm not sure. The plugin mayhaps have an added bonus of allowing discord rich presence?  Not sure. May scrobble to lfm and teal.
